themes/src/main/resources/theme/base/login/messages/messages_en.properties

```diff
@@ -474,6 +474,10 @@ webauthn-error-user-not-found=Unknown user authenticated by the Security key.
 # Identity provider
 identity-provider-redirector=Connect with another Identity Provider
 identity-provider-login-label=Or sign in with
+idp-email-verification-display-name=Email Verification
```

Comment:
What is the motive behind capitalizing `Verification`?

Reply:
Never mind, I just saw that `Verification` is also capitalized in different other translations.